cypress-io/cypress
# chore: add reporter webpack to gulp watch script

Add reporter's webpack watch script to gulp watch. (We should really evaluate the diverging scripts in our repo at some point.)

### Steps to test


1. remove gulp changes
2. update code in reporter src
3. update test in reporter to check for change
4. see test fail
5. reapply gulp change
6. see test pass

### How has the user experience changed?

n/a

better dev expereince

### PR Tasks


- [n/a] Have tests been added/updated?
- [n/a] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
